docs/developer-guide/tests/overview.md

```diff
@@ -0,0 +1,4 @@
+# Tests
+
+## Introduction
+`GreptimeDB` has lots of tests to ensure its behavior. This chapter will introduce several significant methods used to test `GreptimeDB`, and how to work with them.
```

Comment:
```suggestion
Our team has performed different types of testings to ensure the performance of GreptimeDB. This chapter introduces several significant methods used to test `GreptimeDB`, and how to work with them.
```

Reply:
>...the performance of...

The tests we are discussing here are about behavior, i.e., what is allowed and what to expect etc. The performance test usually refers as "bench".